maven/apps/npm-deduplication-portlets/angular/angular-provider/bnd.bnd

```diff
@@ -0,0 +1,10 @@
+Bundle-Name: Angular Provider
+Bundle-SymbolicName: com.liferay.blade.angular.provider
+Bundle-Version: 1.0.0
+-contract: JavaPortlet
```

Comment:
@gamerson I'm not sure we need to add these plugins for this provider module. Can you remove the ones we dont need?